pytest_playwright/pytest_playwright.py

```diff
@@ -159,11 +159,11 @@ def browser_context_args(
     pytestconfig: Any,
     playwright: Playwright,
     device: Optional[str],
+    base_url: Optional[str],
 ) -> Dict:
     context_args = {}
     if device:
         context_args.update(playwright.devices[device])
-    base_url = pytestconfig.getoption("--base-url")
```

Comment:
I don't see where do we read it from the command line anymore. Should we include our default `base_url` fixture implementation?

Reply:
its handled in a third-party package: https://github.com/pytest-dev/pytest-base-url/blob/master/src/pytest_base_url/plugin.py#L11